### PR TITLE
Update documentation for container workflow

### PR DESCRIPTION
**Pull request by martinpitt**
Please let me know if we should go one step further already and stop documenting the manual way. You can still call run_kickstart_tests.sh inside the container, of course, so we can keep that part of the docs.

So let me know your preferences, please!

### CODE REVIEW

**Review by jkonecny12 — APPROVED** [review]
I agree with the current solution. Let's leave it here for some time and we can remove it later.

**Review by rvykydal — APPROVED** [review]
Looks good to me.

File: README.rst [hunk 1]
```diff
@@ -1,22 +1,42 @@
 Kickstart Test Documentation
 ****************************
 
+Kickstart tests are one way of testing the Anaconda Installer, by running an automated installation based on a kickstart file and checking the results.
+
 :Authors:
    Chris Lumens <clumens@redhat.com>
    Martin Kolman <mkolman@redhat.com>
```
Comment (VladimirSlavik) [inline README.rst:8]: This list is missing Martin Pitt, I think?

(Also not sure about @rvykydal's earlier contributions.)

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**